src/com/intellij/idea/plugin/hybris/project/descriptors/YModuleLibDescriptorUtil.kt

```diff
@@ -311,6 +311,24 @@ object YModuleLibDescriptorUtil {
             )
         )
 
+        (descriptor as? YCommonWebSubModuleDescriptor)
```

Comment:
We have `when` switcher for each descriptor 
<img width="780" alt="image" src="https://github.com/epam/sap-commerce-intellij-idea-plugin/assets/2292510/aeef6a4b-81e8-49f2-bda5-417790b9ae2f">

it would be beneficial to introduce new method just for `YCommonWebSubModuleDescriptor`, that method can call  `getWebLibraryDescriptors(descriptor, "Common Web")` at first and then have your new logic